server/.env

```diff
@@ -1 +0,0 @@
-BASE_URL=http://localhost:3000
```

Comment:
Why was this file deleted?

Reply:
I created some dummy data and used it for testing. For this reason, I deleted that file and merged it. I thought it will effect other file.